Assignment3y4_PaulaVelascoLopez

PauVL opened the pull request.

PauVL committed:
Assignment 2b pt1

PauVL committed:
Assignment 2b pt2

PauVL committed:
Assignment3_PaulaVelascoLópez

PauVL committed:
Assignment4_PaulaVelascoLopez

ocorcho commented:
The JSON-LD is not correct as I explained in the classroom.

PauVL committed:
Merge branch 'FacultadInformatica-LinkedData:master' into master

dgarijo commented on Assignment4/PauVL-d21c024/task06.py:
```diff
@@ -0,0 +1,93 @@
+g.add(resource)
+
+#vcard_rdf:Works rdfs:domain ns:Person
+resource = (vcard_rdf.Works,RDFS.domain,ns.Person)
```
you should create new properties on someone else's domain. Use your own namespace

dgarijo commented on Assignment4/PauVL-d21c024/task06.py:
```diff
@@ -0,0 +1,93 @@
+g.add(resource)
+
+#ns:JohnSmith vcard_rdf:Works ns:UPM
+resource = (ns.JohnSmith,vcard_rdf.Works,ns.UPM)
```
same as above

dgarijo commented on Assignment4/PauVL-d21c024/task07.py:
```diff
@@ -0,0 +1,84 @@
+print("7.3")
+for r in g.query(q3):
+  print(r)
+print("FIN")
```
Missing 7.1 .. 7.3 in rdflib

PauVL replied:
Ya he modificado el Assignment 2b y he corregido lo del namespace en el task06 del Assignment 4; pero no se a que te refieres cuando dices que falta el task07, así que lo he vuelto a subir (el task07) poniendo al final de cada apartado print("FIN 7.elapartado") en vez de solo print("FIN") como estaba antes.

dgarijo reviewed (changes requested):
Nicely done, only small changes requested

PauVL committed:
Merge branch 'FacultadInformatica-LinkedData:master' into master

PauVL committed:
Merge branch 'FacultadInformatica-LinkedData:master' into master

PauVL committed:
Delete PauVL-fromJSONtoJSON-LD.jsonld

PauVL committed:
Delete Assignment4/PauVL-d21c024 directory

PauVL committed:
Assignment 2b pt2 modificado

PauVL committed:
Assignment 4 modificado

dgarijo commented on Assignment4/PauVL-d21c024-modificado/task07.py:
```diff
@@ -0,0 +1,85 @@
+    ?X rdf:type ?Subclase.
+    ?X vcard:FN ?A.
+    ?X vcard:Given ?B.
+    ?X vcard:Family ?C.
```
you need to generalize this to obtain all properties, not just these ones.

dgarijo commented on Assignment4/PauVL-d21c024-modificado/task07.py:
```diff
@@ -0,0 +1,85 @@
+print("7.3")
+for r in g.query(q3):
+  print(r)
+print("FIN 7.3")
```
Missing exercise 7.1 .. 7.3 in rdf lib

PauVL committed:
Group03 Assignment 1

PauVL committed:
Delete Assignment4/PauVL-d21c024-modificado directory

PauVL committed:
Add files via upload

Ya he modificado el task07 (los otros tres están como antes, pero he decidido subirlo todo otra vez por si había algún problema)

PauVL committed:
Merge branch 'FacultadInformatica-LinkedData:master' into master

PauVL committed:
Delete HandsOn/Grupo03 directory
